src/uv-common.c

```diff
@@ -33,6 +33,78 @@
 # include <net/if.h> /* if_nametoindex */
 #endif
 
+
+typedef struct {
+    uv_malloc_func local_malloc;
+    uv_realloc_func local_realloc;
+    uv_free_func local_free;
+} uv__allocator_t;
+
+static uv__allocator_t uv__allocator = {
+  malloc,
+  realloc,
+  free,
+};
+
+void* uv__calloc(size_t count, size_t size) {
+  /* Note: the multiplication is not safe if
+   * the result overflows a 'size_t' */
```

Comment:
Obsolete comment?  If you think it still applies, maybe turn the assert into a `if (overflow) return NULL` statement.